src/V3Options.cpp

```diff
@@ -766,6 +770,7 @@ void V3Options::parseOptsList(FileLine* fl, const string& optdir, int argc, char
             else if ( onoff (sw, "-inhibit-sim", flag/*ref*/))  { m_inhibitSim = flag; }
             else if ( onoff (sw, "-lint-only", flag/*ref*/))    { m_lintOnly = flag; }
             else if (!strcmp(sw, "-no-pins64"))                 { m_pinsBv = 33; }
+            else if (!strcmp(sw, "-no-verilate"))               { m_noVerilate = true; }
```

Comment:
Move above as

             else if ( onoff (sw, "-verilate", flag/*ref*/))     { m_verilate = flag; }

and flip the sense of the var.  This is so "--verilate" also works which is more intuitive.

(--no-pins64 is weird as it overrides a more complicated setting.)

Reply:
Thanks, now it is as your suggestion.
https://github.com/verilator/verilator/blob/0f96a4fa7dc72668d641ef18d31665edb9c896f7/src/V3Options.cpp#L811